services/ScheduleSign.proto

```diff
@@ -28,14 +28,24 @@ option java_multiple_files = true;
 import "BasicTypes.proto";
 
 /*
-Provides signatures for a given Scheduled Transaction. The Scheduled Transaction executes if all of the required signatures are witnessed.
-INVALID_SCHEDULE_ID is returned if the specified ScheduleID does not exist; or has now expired, been deleted, or been executed
-UNRESOLVABLE_REQUIRED_SIGNERS is returned if any of the required keys were updated/deleted and cannot be resolved
-INVALID_ACCOUNT_ID is returned if the payerAccountID has been deleted or cannot be resolved
-NO_NEW_VALID_SIGNATURES is returned if there are no new signatures provided in the sigMap to be witnessed
-SOME_SIGNATURES_WERE_INVALID is returned if one of the signatures provided does not represent a valid signature for any required signer
+Adds signing keys to a scheduled transaction. If the new keys satisfy the scheduled transaction's signing
+requirements, and it was scheduled with <tt>executeImmediatelyOnValidSignatures=true</tt>, it will become 
+eligible for execution. In Hedera Services 0.13.0, this will in fact cause its execution immediately
```

Comment:
Should not mention specific versions in case they change.

Reply:
Done.